src/runtime/cmd/kata-runtime/kata-check.go

```diff
@@ -538,3 +648,62 @@ func genericCheckKVMExtensions(extensions map[string]kvmExtension) (map[string]i
 
 	return results, nil
 }
+
+func getVersionsYaml() (ThirdPartyVersions, error) {
+
+	versionFile, err := ioutil.ReadFile("../../versions.yaml")
+
+	var data ThirdPartyVersions
+
+	if err != nil {
+
+		return data, err
+	}
+
+	err2 := yaml.Unmarshal(versionFile, &data)
+
+	if err2 != nil {
+
+		return data, err2
+	}
+
+	return data, nil
+}
+
+func checkVersions() (err error) {
+
+	versionYaml, err := getVersionsYaml()
+
+	if err != nil {
+		return err
+	}
+
+	// Crio is specified by branch/release rather than version in versions.yaml
+	crioVerRec := strings.Join(strings.SplitN(versionYaml.Externals.Crio.Branch, "release-", 2), "")
+	containerdVerRec := versionYaml.Externals.Containerd.Version
+
+	// Extract version of Crio
+	crioVerCmd := exec.Command("crio", "--version")
+	crioVerArr, err := crioVerCmd.Output()
+	crioVer := string(crioVerArr[:])
```

Comment:
`crioVer := string(crioVerArr[:])` may be the same as `crioVer := string(crioVerArr)`